src/Components/DropDown/index.tsx

```diff
@@ -30,17 +30,16 @@ const tamanho = (lugar: string) => {
     {
         case 'cardapio': return '90vw';
     
-        default: return '82.25vw';
+        default: return '80.5vw';
```

Comment:
Ué, tava certo antes
```suggestion
        default: return '82.25vw';
```
Achei que vc tinha compensado em outro lugar, mas ficou diferente da develop msm

Reply:
Não entendi, sei que não mexi.